src/control/cmd/dmg/pretty/pool_test.go

```diff
@@ -972,9 +982,9 @@ no pools in system
 				},
 			},
 			expPrintStr: `
-Pool     Size State Used Imbalance Disabled 
-----     ---- ----- ---- --------- -------- 
-00000001 0 B  Ready 0%   0%        0/0      
+Pool     Size State Used Imbalance Disabled Upgrade 
+----     ---- ----- ---- --------- -------- ------- 
+00000001 0 B  Ready 0%   0%        0/0      0->0    
```

Comment:
In my comment above, I suggested that we shouldn't even show the Upgrade column if none of the pools require an upgrade. If only a subset require an upgrade, we should just show those. My concern with `OK` is that it's a little ambiguous. Does it mean the upgrade went OK? Does it mean it's OK to upgrade? Maybe instead of showing nothing or `OK`, we instead show `None` to indicate that there is no upgrade available/necessary?

Here are a few scenarios, just for visual comparison:

**One pool needs an upgrade, one doesn't:**
```
Pool     Size   State Used Imbalance Disabled Upgrade? 
----     ----   ----- ---- --------- -------- ------- 
00000001 6.0 TB Ready 83%  12%       0/16     1->2    
two      6.0 TB Ready 83%  12%       8/64     None 
```

**No pools need an upgrade (always show column):**
```
Pool     Size   State Used Imbalance Disabled Upgrade? 
----     ----   ----- ---- --------- -------- ------- 
00000001 6.0 TB Ready 83%  12%       0/16     None    
two      6.0 TB Ready 83%  12%       8/64     None 
```

**No pools need an upgrade (selectively show column):**
```
Pool     Size   State Used Imbalance Disabled
----     ----   ----- ---- --------- --------
00000001 6.0 TB Ready 83%  12%       0/16
two      6.0 TB Ready 83%  12%       8/64
```

Personally, I like the second option because it reduces visual clutter. If the admin doesn't need to know about something, we shouldn't show it by default. Note that the `--json` option is always available for full information about the pools. We don't do any editing of information in that mode.

Reply:
In fact, I see your comment after finishing my own review.
From my side either solutions that your proposed are OK for me as they should not confuse the end user, and the json output could be used for admin scripting tasks.